pedometer/chapter.md

```diff
@@ -682,13 +691,15 @@ Let's examine each of these two requirements.
 
 ## 1. Storing and Retrieving Data
 
-Our app needs to store input data to, and retrieve data from, the file system. We'll create an `Upload` class to do this. We'll store the text file containing the data samples directly to the file system, with the file path accessed through the `@file_path` instance variable. The information on the user and trial will be stored in the file name, and we'll have access to those objects through `@user` and `@trial` instance variables. Our `Upload` class has three class-level methods for file system access and retrieval, all of which return one of more instances of `Upload`:
+Our app needs to store input data to, and retrieve data from, the file system. We'll create an `Upload` class to do this. We'll store the text file containing the data samples directly in the file system, with the file path accessed through the `@file_path` instance variable. The information on the user and trial will be stored in the filename, and we'll have access to those objects through `@user` and `@trial` instance variables. 
+
+Our `Upload` class has three class-level methods for file system access and retrieval, all of which return one or more instances of `Upload`:
 
-* `create` takes a file along with user and trial information. It stores the file to the file system, under a file name containing the user and trial information.
-* `find` takes a file path and returns an instance of `Upload`.
+* `create` takes a file along with user and trial information. It stores the file to the file system, under a filename containing the user and trial information.
+* `find` takes a pathname and returns an instance of `Upload`.
 * `all` returns an array of `Upload` instances, one for each accelerometer data file in the file system.
 
-## Separation of Concerns in Upload
+### Separation of Concerns in Upload
```

Comment:
I moved this header down a level since it doesn't make sense to have the two numbered headers at the same level, with an unnumbered header at that same level in between them.